### PR TITLE
Upgrade grpc version to 1.47.0

### PR DESCRIPTION
**Pull request by jaime0815**
Signed-off-by: yun.zhang <yun.zhang@zilliz.com>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 build==0.4.0
 certifi==2021.5.30
 chardet==4.0.0
-grpcio>=1.37.1
-grpcio-testing>=1.37.1
-grpcio-tools>=1.37.1,<=1.43.0
+grpcio==1.47.0
+grpcio-testing==1.47.0
+grpcio-tools==1.47.0
```
Comment (longjiquan) [inline requirements.txt:6]: Please also change these versions in `setup.py`.

Comment (longjiquan) [reply requirements.txt:6]: By the way, any suggestions why did we need this?

**Review by longjiquan — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (jaime0815) [reply requirements.txt:6]: > By the way, any suggestions why did we need this?

https://github.com/grpc/grpc/issues/24069 https://github.com/grpc/grpc/pull/29050

**Pull request merged**